javascript/javascript2/week1/Find and count the Danish letters.js

```diff
@@ -0,0 +1,59 @@
+
+ 
+
+ 
+
+
+
+//first way
+ 
+function countdanishLettersRegex(danishString) {
+
+ let matchingInstances = danishString.match(/[æåø]/gi);
+
+ if(matchingInstances){  
+
+      console.log(`total ${matchingInstances.length}: {${matchingInstances} } `) 
+     return matchingInstances.length
+ }
+ else {
+     return 0
+ }
+}
+const danishString = "Jeg har en blå bil";
+countdanishLettersRegex(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = "Blå grød med røde bær";
+countdanishLettersRegex(danishString2); // returns {total: 4, æ: 1, ø: 2, å: 1}
```

Comment:
Excellent, well done! 👍 
But you forgot to put Semicolon at the end of these lines: 16-17-20